src/gestures.c

```diff
@@ -351,6 +351,13 @@ static void buttons_update(struct Gestures* gs,
 				else if (touching == 3 && cfg->button_3touch > 0)
 					trigger_button_emulation(gs, cfg->button_3touch - 1);
 			}
+			else if (cfg->button_0touch > 0) {
```

Comment:
Since click condition are verified by `integrated_just_clicked` and no other path are true, `cfg->button_0touch > 0` is enough.